website/docs/cdktf/concepts/variables-and-outputs.mdx

```diff
@@ -100,6 +100,14 @@ new TerraformResource(this, "resource", {
 });
 ```
 
+### Passing Input Variables to CDKTF
+
+You can pass input variables e.g. with the name `imageId` to CDKTF in the following ways:
+
+- Environment variable: `TF_VAR_imageId=ami-abcde123`
+- CLI argument: `cdktf deploy --var='imageId=ami-abcde123'` or `cdktf diff --var='imageId=ami-abcde123'` or `cdktf destroy --var='imageId=ami-abcde123'`
```

Comment:
I'd keep it shorter here and note the commands that support these options in the sentence above.
```suggestion
- CLI argument: `cdktf deploy --var='imageId=ami-abcde123'`
```